lib/vssolution.cpp

```diff
@@ -227,22 +227,20 @@ void VisualizationSceneSolution::ToggleDrawBdr()
    // showing them in color (drawbdr == 2) and restore it back when not.
    if (drawbdr == 2)
    {
-      old_minv = minv;
-      old_maxv = maxv;
       minv = 1;
-      maxv = mesh->bdr_attributes.Max();
+      maxv = mesh->bdr_attributes.Size() ? mesh->bdr_attributes.Max() : 1;
+      FixValueRange();
+      UpdateValueRange(true);
+      PrepareBoundary();
    }
    else if (drawbdr == 1)
    {
-      return;
+      PrepareBoundary();
    }
    else if (drawbdr == 0)
    {
-      minv = old_minv;
-      maxv = old_maxv;
+      FindNewValueRange(true);
```

Comment:
This doesn't quite restore the range for me, see e.g. `./glvis -m ../mfem/data/beam-quad.mesh -mslw 6 -oldgl -k "APPPP/////////c"`